src/dependabot/update_metadata.ts

```diff
@@ -1,38 +1,66 @@
 import * as YAML from 'yaml'
 
-export interface updatedDependency {
+export interface dependencyAlert {
+  alertState: string,
+  ghsaId: string,
+  cvss: number
+}
+
+export interface updatedDependency extends dependencyAlert {
   dependencyName: string,
   dependencyType: string,
   updateType: string,
   directory: string,
   packageEcosystem: string,
-  targetBranch: string
+  targetBranch: string,
+  prevVersion: string,
+  newVersion: string,
+  compatScore: number
```

Comment:
I'll need to double check out API to see how we handle Dependency Score `unknown` in the API as I can't remember off the top of my head.